packages/devtools/web/styles_dark.css

```diff
@@ -68,12 +68,17 @@ nav.menu .menu-item {
 
 /* Tweak how buttons look in the dark theme to not be so dark */
 .btn:not(.btn-primary), .btn:not(.btn-primary)[disabled] {
-    background-image: linear-gradient(-180deg, #3f3f3f 0%, #3f3f3f 90%);
+    border: 1px solid #222222;
+    border-radius: 3px;
+    background: linear-gradient(#5f5f5f 5%, #535353 10%, #4f4f4f 100%);
+    box-shadow: 1px 1px 1px rgba(0,0,0,0.2);
+    padding: 3px 10px;
 }
-
 .btn.selected:not(.btn-primary), .btn.selected:not(.btn-primary)[disabled] {
-    background-image: linear-gradient(-180deg, #1f1f1f 0%, #1f1f1f 90%);
-    box-shadow: none;
+    color: #cacbcc;
```

Comment:
This color doesn't visually jump out at me as being selected.
Maybe we should copy the solid color used to indicate that a row is selected in the list of scripts in the debugger view and use that here.
Note that color might also need to change to be more obvious. For example,. consider making it a dark blue or dark orange color to make selection obvious.

Reply:
How about this? I tried to make it more like the blue in the inspector icons:

<img width="1153" alt="Screenshot 2019-03-25 at 2 15 22 pm" src="https://user-images.githubusercontent.com/1078012/54926563-77d08e00-4f08-11e9-844a-473496d472dc.png">

```css
.btn.selected:not(.btn-primary), .btn.selected:not(.btn-primary)[disabled] {
    color: #ffffff;
    background: linear-gradient(#1C547C 5%, #104870 90%, #0C446C 100%);
}
```